scripts/script.yaml

```diff
@@ -64,7 +64,7 @@
                   cmd: fetch_previous_reports                            # <--- Generate the list of options (reporters) to show
                   variable: _report_options
                   params:
-                    - "דיווח חדש ב{{street}} {{city_town}}"
+                    - "דיווח חדש ב{{_location}}"
```

Comment:
Just to make sure - how does location gets its value at this point?

Reply:
You are right, old records are missing the `_location` field.
I choose to set the value as part of fetch_previous_reports, other alternatives seems a bit too complicated.

Reply:
I think that adding this as a new parameter is not necessary as you can simply leave the first parameter as is.
location calculation will be done later on (see below)

Reply:
Yeap, the only difference would be an extra space in case an empty street name but IMO that's is perfectly fine